src/main/resources/view/DarkTheme.css

```diff
@@ -165,14 +165,24 @@
     -fx-background-color: #d9d9d9;
 }
 
-.cell_big_label, .titled-pane, .titled-pane > .title .label {
+.titled-pane, .titled-pane > .title Text {
+    -fx-font-family: "Lato";
+    -fx-font-weight: 700;
+    -fx-font-size: 18px;
+    -fx-fill: #144073;
+}
+
+.titled-pane, .titled-pane > .title .label {
```

Comment:
`Text` and `label` are mainly the same except the last line. Is it possible to extract the front part out as `.titled-pane, .titled-pane > .title` ? May reduce some redundancies

Reply:
I tried that before but it does not work as. The `.title` is actually a `HBox` inside `TitledPane`. So need to select the `Text` and `.label` first. Applying to `HBox` doesn't apply to the child.